app/src/androidTest/java/com/github/bgabriel998/softwaredevproject/ImageActivityTest.java

```diff
@@ -7,26 +7,49 @@
 import androidx.test.espresso.Espresso;
 import androidx.test.espresso.ViewInteraction;
 import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
 
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 
 import static androidx.test.espresso.Espresso.onView;
 import static androidx.test.espresso.action.ViewActions.click;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
 import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static com.github.bgabriel998.softwaredevproject.UITestHelper.AddImageFile;
+import static com.github.bgabriel998.softwaredevproject.UITestHelper.ClearGallery;
 import static com.github.bgabriel998.softwaredevproject.UITestHelper.withDrawable;
+import static com.github.bgabriel998.softwaredevproject.UITestHelper.withImagePath;
 import static org.junit.Assert.assertSame;
+import static org.junit.Assert.fail;
 
+@RunWith(AndroidJUnit4.class)
 public class ImageActivityTest {
 
-    private static final int testImage = R.drawable.temp_camera;
+    private static final int testImage = R.drawable.temp_diablerets;
+    private static final String imagePath = CameraActivity.getOutputDirectory(ApplicationProvider.getApplicationContext()) +
+                                                "/TestImage";
 
     static Intent intent;
     static {
         intent = new Intent(ApplicationProvider.getApplicationContext(), ImageActivity.class);
-        intent.putExtra("image", testImage);
+        intent.putExtra("imagePath", imagePath);
```

Comment:
```suggestion
        intent.putExtra(ImageActivity.IMAGE_PATH_INTENT, imagePath);
```
Discussion on discord!